pipeline/pipeline-notes.rst

```diff
@@ -22,37 +22,56 @@ Make sure you edit your security groups to include port 22 (SSH) and port
 80 (HTTP) ; you'll need the first one to log in, and the second one to 
 connect to the ipython notebook.
 
-Once you ssh in, set yourself up to run as root.::
-
- sudo su
+If you are running on RackSpace, we will need to add extra space for data by formating and mounting a drive.
+Formating/Mounting Drive::
+  mkfs -t ext4 /dev/xvde1
+  mkdir /mnt/data
+  mount /dev/xvde1 /mnt/data/ -t ext4
+  # To make a regular user 
+  adduser non-root # TODO find quick setup instructions
 
 Next, we are going to set the instance up with many of the software 
-packages we will need::
-
+packages we will need. You will need root permissions to install these::
 
- apt-get update
- apt-get --yes install screen git curl gcc make g++ python-dev unzip \
+ sudo apt-get update
+ sudo apt-get --yes install screen git curl gcc make g++ python2.7-dev unzip \
             default-jre pkg-config libncurses5-dev r-base-core \
             r-cran-gplots python-matplotlib sysstat bowtie \
             texlive-latex-recommended mummer python-pip ipython \
-            ipython-notebook bioperl ncbi-blast+
+            ipython-notebook bioperl ncbi-blast+ \
+            python-virtualenv 
+
+Non-EC2::
+  mkdir ~/bin
+
+If running on EC2::
+ sudo  mkdir /mnt/bin
+ sudo chown ubuntu /mnt/bin
+ ln -s /mnt/bin ${HOME}/bin
```

Comment:
I think these lines need more indention to be a literal quote; see https://github.com/ged-lab/2012-paper-diginorm/blob/non-root-install/pipeline/pipeline-notes.rst